### ISSUE TITLE
Problem about "perf-factor" hyper-parameter for 2080ti

### ISSUE THREAD

**Issue by UCAS-GYX** [karthiksharma98/sap-starterkit#1]
Following the  instructions in this guide, i obtain this baseline on a 2080ti GPU, for Mask RCNN FPN R50 with input scale 1.0, using given configs and pretrained weigth, **start_toolkit.sh** is ran with default settings followed by **start_det_baseline.sh** and results are as follows:
```
Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.130
Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.272
Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.112
Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.045
Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.094
Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.193
Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.098
Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.225
Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.233
Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.100
Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.220
Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.329
```

To provide a runtime scaling factor to compensate for difference in hardware,i set **perf-factor** as 0.8324324 which is suggested in this guide. However, the worse result is got:
```
Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.042
Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.077
Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.041
Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.012
Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.021
Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.061
Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.031
Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.096
Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.099
Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.036
Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.079
Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.137
```

There are my specific settings in start_toolkit.sh

```
dataDir="/home/server/GYX/sap/data"
methodName=mrcnn50_nm
scale=1.0
factor=0.8324324
# "nm" is short for "no mask"

python -m sap_toolkit.server \
        --data-root "$dataDir/Argoverse-1.1/tracking" \
        --annot-path "$dataDir/Argoverse-HD/annotations/val.json" \
        --overwrite \
        --out-dir "$dataDir/Exp/Argoverse-HD/output/${methodName}_s${scale}/val" \
        --eval-config "./config.json" \
        --perf-factor $factor
```

**Comment by karthiksharma98** [comment]
Hi, thanks for reaching out with your issue. We are looking into it and will get back to you as soon as possible.

**Comment by karthiksharma98** [comment]
This issue has now been resolved. Please update your `sap_toolkit` version to 1.0.0 by using the command `pip install sap_toolkit --upgrade`. The scripts `start_det_baseline.sh` and `start_fs_baseline.sh` have also been updated and now require the `--perf-factor` argument. Please `git pull` to pull the corresponding changes.
